### PR TITLE
[14.0][OU-ADD] l10n_be_invoice_bba: nothing to do

### PR DESCRIPTION
**Pull request by remytms**
Module is left empty. Code was moved to l10n_be.

### CODE REVIEW

File: docsource/modules130-140.rst [hunk 1]
```diff
@@ -236,7 +236,7 @@ Module coverage 13.0 -> 14.0
 +--------------------------------------------+-------------------------------------------------+
 |l10n_be_edi                                 |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-|l10n_be_invoice_bba                         |                                                 |
+|l10n_be_invoice_bba                         | Nothing to do. Merged into l10n_be              |
```
Comment (MiquelRForgeFlow) [inline docsource/modules130-140.rst]:
```suggestion
|l10n_be_invoice_bba                         | Merged into l10n_be                             |
```
l10n_be has not been handled yet.

**Comment by MiquelRForgeFlow** [comment]
Depends on https://github.com/OCA/OpenUpgrade/pull/4308.

**Review by robinkeunen — APPROVED** [review]
lgtm

File: docsource/modules130-140.rst [hunk 2]
```diff
@@ -236,7 +236,7 @@ Module coverage 13.0 -> 14.0
 +--------------------------------------------+-------------------------------------------------+
 |l10n_be_edi                                 |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-|l10n_be_invoice_bba                         |                                                 |
+|l10n_be_invoice_bba                         | Merged into l10n_be                             |
```
Comment (StefanRijnhart) [inline docsource/modules130-140.rst]: Strictly speaking, the module should be considered merged for our purposes only in 15.0, as the module is still present in the Odoo 14 distribution and installable. So I'd put `Nothing To Do` here.

Comment (remytms) [reply docsource/modules130-140.rst]: Fixed.

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**